client/overview/task-list/tasks/reconnect-task.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { TaskItemProps } from '../types';
+
+export const getReconnectWpcomTask = (
+	wpcomReconnectUrl: string
+): TaskItemProps | null => {
+	const handleClick = () => {
+		window.location.href = wpcomReconnectUrl;
+	};
+
+	return {
+		key: 'reconnect-wpcom-user',
+		level: 1,
+		content: '',
+		title: __( 'Reconnect WooCommerce Payments', 'woocommerce-payments' ),
```

Comment:
```suggestion
		title: sprintf( 
		    /* translators: %s: WooPayments */
		    __( 'Reconnect %s', 'woocommerce-payments' ), 'WooPayments' ),
```

Reply:
Thanks. These got lost in the merge because of file name changes. All WooPayments branding rollout fixes should be made in https://github.com/Automattic/woocommerce-payments/pull/6696/commits/3b86678ce0fbb9abb4edd26580429330d6f3a8da. 🙂